packages/kbn-apm-config-loader/src/config.ts

```diff
@@ -143,6 +144,15 @@ export class ApmConfiguration {
       config.active = true;
     }
 
+    if (process.env.ELASTIC_APM_KIBANA_FRONTEND_ACTIVE === 'false') {
```

Comment:
@jbudz Just to get some context, was this the issue that we were chatting internally around serverless?

Reply:
Yep, we want to disable RUM.